frontend/src/components/chrome/chrome-sidebar.tsx

```diff
@@ -130,7 +131,7 @@ function SidebarHeader({ close }: SidebarHeaderProps) {
         style={{ fontSize: 14 }}
         data-testid='log-out'
       >
-        Log out
+        Wallets
```

Comment:
This change seems odd?

Reply:
Someone wanted to change this text but reverting for now.